### PR TITLE
Spelling and grammar fixes

### PR DESCRIPTION
**Pull request by moshekaplan**
Mostly found with:
`find . -type f -name "*.c" -exec aspell --mode=ccpp check {} \;`

Common replacements made with:
`grep -rl 'writable' ./ | xargs sed -i 's/writeable/writable/g'`

### CODE REVIEW

**Review by fenner — CHANGES_REQUESTED** [review]
Did you miss the `commited` examples in mib2c-conf.d/parent-set.m2i and mfd-interface.m2c?

There are several instances of `writeable` in local/ in net-snmp-cert and in mib2c-conf.d

In general the source of these repeated consistent typos is a typo in the templates, so updating the templates would help prevent future instances

**Comment by moshekaplan** [comment]
@fenner : I did not review all files. That's why there are probably still plenty of typos remaining. I figured that some improvement is better than nothing :)

File: agent/mibgroup/Rmon/rows.c [hunk 1]
```diff
@@ -67,7 +67,7 @@ rowapi_delete(RMON_ENTRY_T * eold)
     }
 
     /*
-     * delete it in users's sence 
+     * delete it in users's sense
```
Comment (bvanassche) [inline agent/mibgroup/Rmon/rows.c:70]: Although I'm not a native speaker: this doesn't look right to me.

Comment (moshekaplan) [reply agent/mibgroup/Rmon/rows.c:70]: sense is probably correct. It should be user's though.

Comment (moshekaplan) [reply agent/mibgroup/Rmon/rows.c]: Added as an additional commit.

**Comment by bvanassche** [comment]
Thanks for the patches!

**Pull request merged**